src/app/assets/Icons/PrimPrioIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+export default function PrimPrioIcon(): JSX.Element {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="50"
+      height="50"
+      viewBox="0 0 50 50"
+    >
+      <g transform="translate(-811 120)">
+        <g
+          transform="translate(811 -120)"
+          fill="#b04e72"
+          stroke="#b04e72"
+          stroke-width="1"
+        >
+          <circle cx="25" cy="25" r="25" stroke="none" />
+          <circle cx="25" cy="25" r="24.5" fill="none" />
+        </g>
+        <text
+          transform="translate(836 -87)"
+          fill="#fff"
+          font-size="20"
+          font-family="Roboto-Bold, Roboto"
```

Comment:
Roboto and Roboto-Bold are not loaded, consider importing it from https://fonts.google.com/specimen/Roboto